src/Microsoft.AspNet.Mvc.TagHelpers/EnvironmentTagHelper.cs

```diff
@@ -74,36 +68,28 @@ public override void Process(TagHelperContext context, TagHelperOutput output)
                 return;
```

Comment:
Not really related to this PR but this `return` is for a weird case. Very odd that `<environment Names="dev">content</environment>` writes out `content` when `EnvironmentName` is `string.Empty` but not when the property has any other value (but `"dev"`).